heat/core/dndarray.py

```diff
@@ -1702,6 +1699,30 @@ def item(self):
         """
         return self.__array.item()
 
+    def kurtosis(self, axis=None, unbiased=True, Fischer=True):
```

Comment:
Can we have parameter and return type annotations here (except DNDarray due to aliasing)